<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>960</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FCEmulator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="StartMenu">
    <property name="title">
     <string>Start</string>
    </property>
    <addaction name="ActionChooseFile"/>
    <addaction name="actionReload"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="AboutMenu">
    <property name="title">
     <string>Debugger</string>
    </property>
    <addaction name="actionOpenDebugger"/>
   </widget>
   <widget class="QMenu" name="menuSelect">
    <property name="title">
     <string>Option</string>
    </property>
    <addaction name="actionOpenSound"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
   </widget>
   <addaction name="StartMenu"/>
   <addaction name="menuSelect"/>
   <addaction name="AboutMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="ActionChooseFile">
   <property name="text">
    <string>ChooseFile</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="text">
    <string>暂停游戏</string>
   </property>
  </action>
  <action name="actionContinue">
   <property name="text">
    <string>继续游戏</string>
   </property>
  </action>
  <action name="actionOpenDebugger">
   <property name="text">
    <string>OpenDebugger</string>
   </property>
  </action>
  <action name="actionOpenSound">
   <property name="text">
    <string>TurnOnSound</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="text">
    <string>Reload</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
